### ISSUE TITLE
Question about the `DisableForceMultipart`

### ISSUE THREAD

**Issue by terasum** [imroc/req#333]
I'm curious about :
https://github.com/imroc/req/blob/d0373d27487a0fac2a5005e258747bcccb5e44e8/request.go#L1105

```
// DisableForceMultipart disables force using multipart to upload form data.
func (r *Request) DisableForceMultipart() *Request {
	r.isMultiPart = true
	return r
}
```

maybe `r.isMultiPart = flase` is correct?

**Comment by imroc** [comment]
Nice catch! Will be fixed later.

**Comment by imroc** [comment]
It's been fixed in v3.43.1
